prov/util/src/util_mr_map.c

```diff
@@ -257,9 +257,11 @@ void ofi_mr_update_attr(uint32_t user_version, uint64_t caps,
 	if (caps & FI_HMEM) {
 		cur_abi_attr->iface = user_attr->iface;
 		cur_abi_attr->device = user_attr->device;
+		cur_abi_attr->hmem_data = user_attr->hmem_data;
```

Comment:
I think you need to check version here. Only when user_version >= 1.19, can you access `user_attr->hmem_data`.

Reply:
It that necessary(or does it work that way)? `hmem_data` is defined in `fi_domain.h` and we shouldn't expect a version mismatch between these files right?

Reply:
You have to deal with compiled apps which may be against the older header.

Reply:
I see. That makes sense. Good catch!

Reply:
@wzamazon Do you have other comments?